src/main/kotlin/com/dansplugins/factionsystem/MedievalFactions.kt

```diff
@@ -238,6 +232,64 @@ class MedievalFactions : JavaPlugin() {
         )
         setupRpkLockService()
 
+        val metrics = Metrics(this, 8929)
+        metrics.addCustomChart(
+            SimplePie("language_used") {
+                config.getString("language")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("database_dialect") {
+                config.getString("database.dialect")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("average_claims") {
+                factionService.factions
+                    .map {
+                        claimService.getClaims(it.id).size
```

Comment:
I'm wondering whether it would be worth getting the database to do this work.

Reply:
Maybe! I don't imagine it will add too much strain to the server startup process but who knows?